internal/request/valid_outbound.go

```diff
@@ -18,16 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
```

Comment:
I was trying to avoid an overly long file name but it doesn't really matter. Will rename.